<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QMainWindow" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>单词搜索</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(81, 81, 81);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>单词搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWordBaseInfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>未选择搜索单词库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设置单词库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>使用当前模式提供的单词库</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>601</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResult0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>abnormal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResult1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>abnormal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResult2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>abnormal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResult3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>abnormal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResult4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>500</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>abnormal</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearchInfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>没有搜索结果......</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
